### DISCUSSION TITLE
Is there any plans to support localization?

### DISCUSSION THREAD

**Discussion by daxijiu** [comfyanonymous/ComfyUI#981] [General]
The workflow of ComfyUI is very suitable for professional use, and there are many people in China who would like to use ComfyUI. However, english is a barrier. 
Although I am not familiar with programming, I can provide Chinese localization translation if there are any plans for it.

**Comment by reaper47** [comment]
Thank you for your willingness to provide Chinese translations! I've begun work on implementing the localization feature. However, it's currently stalled because I'm facing difficulty in integrating translations into Litegraph without modifying the Litegraph copy ComfyUI uses.

We can either implement native localization support in the Litegraph repository and open a PR or integrate it directly in Comfy's Litegraph copy. Perhaps the community has an idea how to tackle this problem.

**Comment by daxijiu** [comment]
Thank you for your reply. 
I also think it's a good idea not to modify the Litegraph copy.
I'll keep an eye on the progress.

**Reply by reaper47** [reply]
Here is the branch for reference: https://github.com/reaper47/ComfyUI/tree/locales. It might take some time to find a way around the Litegraph block.
